File: CSharp/Supermarket/Model/Checkout/TaxLine.cs
namespace Supermarket.Model.Checkout
{
    public class TaxLine
    {
        public TaxLine(int rate, decimal amount)
        {
            Rate = rate;
            Amount = amount;
        }
        public int Rate { get; }
        public decimal Amount { get; }

        public override string ToString() => $"VAT {Rate/100.0:P0}: {Amount:C}";
    }
}

File: java/src/main/java/supermarket/model/checkout/TaxLine.java
package supermarket.model.checkout;

import java.math.BigDecimal;

public class TaxLine {
	private int rate;
	private BigDecimal amount;

	public int getRate() {
		return rate;
	}

	public void setRate(int rate) {
		this.rate = rate;
	}

	public BigDecimal getAmount() {
		return amount;
	}

	public void setAmount(BigDecimal amount) {
		this.amount = amount;
	}

	@Override
	public String toString() {
		return String.format("VAT %i: %d", getRate() / 100, amount);
	}

}
